README.md

```diff
@@ -460,6 +460,12 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 | :----: | :-------: | ---- | :-----: | :---- |
 | :green_heart:  | STM32WB55CE | [SharkyMKR](https://midatronics.com/shop/development-boards/mkr-sharky-i/) | *1.7.0* |  |
 
+### [elecgator](https://www.elecgator.com/) boards
+
+| Status | Device(s) | Name | Release | Notes |
+| :----: | :-------: | ---- | :-----: | :---- |
+| :green_heart:  | STM32F746ZET | [EtherCATduino](https://www.elecgator.com/products/info/ethercatduino) | *2.1.0* |  |
+
```

Comment:
Please move it to ESC boards section, as it seems to be an ESC board. we would not add a dedicated section.